package.json

```diff
@@ -11,6 +11,7 @@
     "prop-types": "^15.8.1",
```

Comment:
- [ ] You didn't add any license file in your repository. Kindly add a license file in your repo using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush:
- [ ] After adding a license file in your repository, It would be very nice if you would add link to your license file in your readme file to make it more descriptive :+1:

Reply:
Hi @HAMMAS-SALEEM, Thanks for the review but I believe if I am not interested to have a license for my read file it would be okay am bothered to check in for a reduction of my tickets for code review

Reply:
Hey @Kirabo19 I hope you are doing well.
Having a license file is in your project requirements. If you check your requirements where it says Documente work in a professional way then there you will find requirement regarding readme file. In that readme file you will see there is a link to the license file so it means it should be there and you need to a license file to make it work.

Why you need a license file to make your project licensed to you. If you think that this project has not been built by you then you don't need to have a license file. But if you have created this project then you should license this project to you. License works a copyright so nobody can claim this project to be one's project.

I hope it has cleared your doubts with having a license file. If you still wanna discuss it then you can reach out to me on slack by searching my name **Hammas** in **full-time-students** channel :blush:

If you think that this review was unfair then you can ask for a second opinion from senior code reviewers :+1:

Reply:
Hi @HAMMAS-SALEEM, thanks delightful detailed information as per review i have made desired changes